src/Annotation/ApiFilter.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Annotation;
+
+use ApiPlatform\Core\Api\FilterInterface;
+use ApiPlatform\Core\Exception\InvalidArgumentException;
+
+/**
+ * Filter annotation.
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ *
+ * @Annotation
+ * @Target({"PROPERTY", "CLASS"})
```

Comment:
Not sure that `METHOD` target's are a good thing here. Also, they add complexity.